pkg/reconciler/common/transformers.go

```diff
@@ -56,3 +56,18 @@ func Transform(ctx context.Context, manifest *mf.Manifest, instance base.KCompon
 	*manifest = m
 	return nil
 }
+
+// InstalledTransform will mutate the namespace of all installed resources
+func InstalledTransform(manifest *mf.Manifest, instance base.KComponent, extra ...mf.Transformer) error {
```

Comment:
Since this is just injecting the namespace to `manifest`, the function name `InstalledTransform` should be `injectNamespace` or something?